Mathlib/Analysis/SpecialFunctions/Log/PosLog.lean

```diff
@@ -0,0 +1,160 @@
+/-
+Copyright (c) 2025 Stefan Kebekus. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Stefan Kebekus
+-/
+import Mathlib.Analysis.SpecialFunctions.Log.Basic
+
+/-!
+# The Positive Part of the Logarithm
+
+This file defines the function `log⁺ = r ↦ max 0 (log r)`, and establishes
+standard estimates.
+-/
```

Comment:
Can you mention the notation in the file-level docstring, and also maybe `posLog_prod` and `posLog_sum`?